js/components/frame.js

```diff
@@ -877,10 +887,27 @@ class Frame extends ImmutableComponent {
           method = (currentDetail, originalDetail) =>
             windowActions.setAutofillCreditCardDetail(currentDetail, originalDetail)
           break
+        case messages.TORRENT_MESSAGE:
+          // Relay torrent IPC from the webview to a browser process
```

Comment:
I don't think this relaying is necessary; the webview can just call `ipc.send` to send a message to the browser process. See `messages.CERT_ERROR_ACCEPTED` for instance.